<launch>
    <arg name="cinfo" default="/stereo/left/camera_info" />
    <arg name="image" default="/stereo/left/image" />

    <node pkg="blort_ros" type="learnsifts" name="blort_learnsifts" args="$(find blort_ros)" output="screen">
        <remap from="/blort_camera" to="$(arg cinfo)" />
        <remap from="/blort_image" to="$(arg image)" />
    </node>

</launch>

